app/models/conference.rb

```diff
@@ -81,6 +82,21 @@ def user_registered? user
   end
 
   ##
+  # Delete all EventSchedules that are not in the hours range
+  # After the conference has been successfully updated
+  def delete_event_schedules
+    if self.start_hour_changed? || self.end_hour_changed?
+      event_schedules = EventSchedule.select do |e|
```

Comment:
Thanks @Ana06 and sorry I didn't knew this thing that's why didn't get it first time. I have made the changes. :wink: